src/title_screen.c

```diff
@@ -64,7 +64,22 @@ static const u32 sTitleScreenRayquazaTilemap[] = INCBIN_U32("graphics/title_scre
 static const u32 sTitleScreenLogoShineGfx[] = INCBIN_U32("graphics/title_screen/logo_shine.4bpp.lz");
 static const u32 sTitleScreenCloudsGfx[] = INCBIN_U32("graphics/title_screen/clouds.4bpp.lz");
 
-const u16 gIntroWaterDropData[] =
+// This is BLDALPHA data. It's equivalent to:
```

Comment:
can you not use this macro here then?

Reply:
I can; but expanding it out would be 32 lines, so I wasn't sure Shanty/Diego would be happy with that.

Reply:
Well _I_ would prefer the macro. Also, I know in modern C you can initialize ranges like so:
```
int array[] = {
    [1] = 0x0110,
    [2 ... 100] = 0x1234,
    [101] = 0x1000,
    ....
}
```
which you can probably use with agbcc to make the end of that array a single line